ui/mirage/utils.js

```diff
@@ -26,3 +24,447 @@ export function arrToObj(prop, alias = '') {
+        ModifyTime: 1647900825385587000,
+        Namespace: 'default',
+        NextEval: 'ea2239aa-26d6-8874-8c56-e1600585772b',
+        NodeID: '',
+        PreviousEval: 'b37d06e4-4eb4-b29d-3b4a-b0c7bf2528ad',
+        Priority: 50,
+        Status: 'failed',
+        StatusDescription: 'evaluation reached delivery limit (3)',
+        TriggeredBy: 'failed-follow-up',
+        Type: 'service',
+        WaitUntil: null,
+      },
+      {
+        BlockedEval: '',
+        CreateIndex: 114,
+        CreateTime: 1647900825385584000,
+        DeploymentID: '',
+        ID: 'ea2239aa-26d6-8874-8c56-e1600585772b',
+        JobID: 'example',
+        ModifyIndex: 128,
+        ModifyTime: 1647900979511304000,
+        Namespace: 'default',
+        NextEval: '25a2dd19-8d22-d1dd-280a-79860c9b8bdb',
+        NodeID: '',
+        PreviousEval: 'd7c50aa5-5bf1-5119-d7e7-0d0ae5381856',
+        Priority: 50,
+        Status: 'failed',
+        StatusDescription: 'evaluation reached delivery limit (3)',
+        TriggeredBy: 'failed-follow-up',
+        Type: 'service',
+        WaitUntil: null,
+      },
+      {
+        BlockedEval: '',
+        CreateIndex: 128,
+        CreateTime: 1647900979511301000,
+        DeploymentID: '',
+        ID: '25a2dd19-8d22-d1dd-280a-79860c9b8bdb',
+        JobID: 'example',
+        ModifyIndex: 136,
+        ModifyTime: 1647901211369652000,
+        Namespace: 'default',
+        NextEval: '1fded690-20ad-6afa-3b89-59e319dfce18',
+        NodeID: '',
+        PreviousEval: 'ea2239aa-26d6-8874-8c56-e1600585772b',
+        Priority: 50,
+        Status: 'failed',
+        StatusDescription: 'evaluation reached delivery limit (3)',
+        TriggeredBy: 'failed-follow-up',
+        Type: 'service',
+        WaitUntil: null,
+      },
+      {
+        BlockedEval: '',
+        CreateIndex: 136,
+        CreateTime: 1647901211369648000,
+        DeploymentID: '',
+        ID: '1fded690-20ad-6afa-3b89-59e319dfce18',
+        JobID: 'example',
+        ModifyIndex: 136,
+        ModifyTime: 1647901211369648000,
+        Namespace: 'default',
+        NextEval: '',
+        NodeID: '',
+        PreviousEval: '25a2dd19-8d22-d1dd-280a-79860c9b8bdb',
+        Priority: 50,
+        Status: 'pending',
+        StatusDescription: '',
+        TriggeredBy: 'failed-follow-up',
+        Type: 'service',
+        WaitUntil: null,
+      },
+    ],
+    Status: 'failed',
+    StatusDescription: 'evaluation reached delivery limit (3)',
+    TriggeredBy: 'failed-follow-up',
+    Type: 'service',
+    Wait: 20000000000,
+  };
+};
+
+export const MOCK_EVALUATION = {
+  CreateIndex: 20,
+  CreateTime: 1647899150314738000,
+  ID: 'fede162c-26a6-c108-178b-1c140f9f5680',
+  JobID: 'example',
+  JobModifyIndex: 10,
+  ModifyIndex: 31,
+  ModifyTime: 1647899318007569000,
+  Namespace: 'default',
+  NextEval: 'fd1cd898-d655-c7e4-17f6-a1a2e98b18ef',
+  PreviousEval: 'd8a5c14f-120a-3d83-6305-90927356dd6c',
+  Priority: 50,
+  RelatedEvals: [
```

Comment:
Yes! I totally want to get good fixtures via `nomad operator debug` or we can model via Mirage. I prefer the prior because its more declarative and won't interfere with our set-up.